### PR TITLE
Adapt translations to Lokalise workflow

### PR DESCRIPTION
**Pull request by osmaczko**
- [chore(translations): fix qml_en.ts locale](https://github.com/status-im/status-desktop/commit/5564f45bf7b0f553a4ef6e84d2a459f5125c71ec) 
    - Lokalise expects *_en files to be en, not en_US.
- [chore(translations): remove *.ts files other than qml_en.ts](https://github.com/status-im/status-desktop/commit/b3d6b92a687291590c754b949567ff440324c28d) 
   - Current Lokalise workflow is:  
  1) Upload strings to translate (qml_en.ts)  
  2) Translate strings to selected languages (this is driven by Lokalise
   configuration)  
  3) Export *.ts files with translations (e.g. qml_de.ts, qml_es.ts)
- [chore(translations): update qml_en.ts file](https://github.com/status-im/status-desktop/commit/246fc6eecc409111df1ff55f1298e3fb0d4579db)

### CODE REVIEW

**Comment by anastasiyaig** [comment]
@osmaczko hey, what is the status of this?

**Comment by anastasiyaig** [comment]
@iurimatias can we please move it from Code Review and QA somewhere else until it is ready?

**Comment by osmaczko** [comment]
> @osmaczko hey, what is the status of this?

@anastasiyaig it was waiting for review (approval), in the meantime I was on holiday. @iurimatias @jrainville could you please take a look?

**Review by jrainville — APPROVED** [review]

**Review by iurimatias — APPROVED** [review]

**Pull request merged**
